src/tools/prun/prun.1.md

```diff
@@ -412,25 +238,25 @@ To manage files and runtime environment:
     directory of the remote machines where processes will be launched
     prior to starting those processes.
 
-`-set-cwd-to-session-dir, --set-cwd-to-session-dir`
+`--set-cwd-to-session-dir`
 
 :   Set the working directory of the started processes to their session
     directory.
 
-`-wd <dir>`
-
-:   Synonym for `-wdir`.
+`--wdir <dir>`
 
-`-wdir <dir>`
-
-:   Change to the directory <dir> before the user's program executes.
+:   Change to the directory `<dir>` before the user's program executes.
     See the "Current Working Directory" section for notes on relative
-    paths. **Note:** If the `-wdir` option appears both on the command
+    paths. **Note:** If the `--wdir` option appears both on the command
     line and in an application context, the context will take precedence
-    over the command line. Thus, if the path to the desired wdir is
+    over the command line. Thus, if the path to the desired working directory is
     different on the backend nodes, then it must be specified as an
     absolute path that is correct for the backend node.
 
+`--wd <dir>`
+
+:   Synonym for `--wdir`.
+
 `-x <env>`
```

Comment:
Sounds like CLI details are in flux and tricky with OMPI/PRRTE separation (#795), but a simple "requires `--personality ompi`" in the description for `-x` here might save a lot of pain.

Reply:
Interesting. @rhc54 is `-x` intended to only be for the OMPI personality? It seems like something that should be part of the core.

Reply:
I now see the discussion [here](https://github.com/openpmix/prrte/pull/795#issuecomment-790953094) and filed Issue #801 to fix the code.